### ISSUE TITLE
No longer compatible with php7.4

### ISSUE THREAD

**Issue by jamesliobjective** [camilord/utilus#2]
Hey, with union return types introduced here:

https://github.com/camilord/utilus/commit/c3e99308fed5d63476d0bb9af11a5f71ad403654
"public function processUpload($file_element): bool|FileElement"

It is no longer compatible with php7.4.  Union types are a php 8 feature, but the composer.json still says  "php": ">=7.4",

**Comment by camilord** [comment]
yeah.. made a mistake on that.. reverting the changes. making it backward compatibility now.
